cypress/e2e/cypress_spec.cy.ts

```diff
@@ -0,0 +1,16 @@
+// This file tests the cypress library is working correctly
```

Comment:
I don't think we need this. Can we remove this file?

Reply:
I thought it could be useful in helping to diagnose failing tests where the cause could be the testing library itself.

But yes, it can be removed.